src/debug/controllers/UserController.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace craft\debug\controllers;
+
+class UserController extends \yii\debug\controllers\UserController
+{
+}
```

Comment:
@timkelty Is this necessary?

Reply:
I think so, unfortunately. Because we're overriding the `$controllerNamespace` so that we can tweak `DefaultController`, we have to also bring this along, otherwise the controller won't get recognized. Unless you know another way?